docker/Makefile

```diff
@@ -0,0 +1,18 @@
+TAG ?= $(shell git rev-parse --short HEAD)
```

Comment:
Not sure if we should tag the image with a commit ID.

Reply:
this is a requirement from issue #7